### ISSUE TITLE
Tutorials in Context

### ISSUE THREAD

**Issue by pelikhan** [microsoft/pxt#5646]
## Description

Start and stop a (micro-)tutorial within the code editor without having to start a new project or loosing your code.

## How to use it?

Click on the **Tutorials** button in the toolbox and select a tutorial.

If you cannot see that button and you editor supports this experiment, go to **About**, then click **Experiments** and **Tutorials in Context**. If you cannot see this card, your editor does not support yet this feature.

## Feedback

Do you use this feature? Is there something you'd like to change? Did you encounter issues? Please drop a comment on this issue to send us your feedback.

**Comment by austinmsuarez** [comment]
I think its useful! If possible it would make a great experience when the user finishes the tutorial you end up back at the project you first started the mini-tutorial or incorporate the code created into the current project. That way it doesn't appear that code has disappeared after the tutorial. I would definitely showcase it more then!

**Comment by pelikhan** [comment]
Yes the idea is that it starts wherever you are and ends where you left the tutorial

**Comment by jwunderl** [comment]
Looks like that might have a bug at the moment, just tried it and it loaded the tutorial with an empty on start.

**Comment by austinmsuarez** [comment]
@pelikhan Yea that would be awesome then! 

@jwunderl For me it keeps whatever was created in the mini-tutorial. The user would then back out to projects and select their project they were working on.
